tempalte4.yaml

```diff
@@ -0,0 +1,3 @@
+{
+"awsTokens":"segthwsfn324dsfv"
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Code Generic   
**Description:** Code Generic Credential   
**Severity:** HIGH   
**Match:** "awsTokens":"****************"  
[This comment was created by Aqua Pipeline]